src/index.tsx

```diff
@@ -8,3 +8,4 @@ export { default as Avatar, AvatarProps } from './Avatar';
 export { default as AvatarGroup, AvatarGroupProps } from './AvatarGroup';
 export { default as Badge, BadgeProps } from './Badge';
 export { default as Popover } from './Popover';
+export { default as Input, InputProps, SearchProps } from './Input'
```

Comment:
Need to re-export everything in `/Input/index.tsx`. The `Search` component is not exported here